tutorials/javaFxPart2.md

```diff
@@ -22,6 +22,8 @@ Mockup | Control
 ![TextField](images/javafx/MockupTextField.png) | TextField
 ![ScrollPane](images/javafx/MockupScrollPane.png) | ScrollPane
 
+{{ icon_tip }} You may be wondering why a ScrollPane is a `Control`, and not a `Pane`, as its name suggests. `ScrollPane` extends the `Control` class in JavaFX, because it represents a UI "widget" which the user directly interacts with (such as `Label`, `Button`, and `TextField`), whereas a `Pane` is a mere container that contains the other components. Of course, ScrollPane has the functionality of a container, allowing it to contain, or rather show a viewport of, its content.
+
```

Comment:
@laney0808 Do readers know what a viewport is, by this point? If not, you can use a tooltip to give a definition.
```
<tooltip content="i.e., ...">viewport</tooltip>
```

Reply:
Sure!

Reply:
The term `window` has different meanings, which might confuse the reader.